rent.go

```diff
@@ -15,6 +15,10 @@ const (
 	VByteCostFactorData VByteCostFactor = 1
 	// VByteCostFactorKey defines the multiplier for fields which can act as keys for lookups.
 	VByteCostFactorKey VByteCostFactor = 10
+	// VByteCostFactorBlockIssuerKeys defines the multiplier for block issuer public keys.
+	VByteCostFactorBlockIssuerKeys VByteCostFactor = 10
+	// VByteCostFactorStakingFeature defines the multiplier for account output's staking feature.
+	VByteCostFactorStakingFeature
```

Comment:
we should remove this constants because they should always come from the protocol parameters